src/pybind/mgr/dashboard/controllers/docs.py

```diff
@@ -124,14 +122,14 @@ def _add_param_info(cls, parameters, p_info):
                 for parameter in parameters:
                     if p['name'] == parameter['name']:
                         parameter['type'] = p['type']
-                        parameter['description'] = p['description']
+                        parameter['description'] = p['description'] or "*None*"
```

Comment:
Minor tip: it's a good practice to replace repeated literals (`*None*`, `*No description available*`) with variables
```suggestion
# global scope
NO_DESCRIPTION_AVAILABLE = "*No Description Available*"
...

                        parameter['description'] = p['description'] or NO_DESCRIPTION_AVAILABLE
```

Reply:
Added in global scoped string for missing descriptions: implemented in the fix function

Reply:
Could you please replace all usages?

Reply:
Thanks!

Reply:
Everything should be replaced now!